packages/router/src/util.ts

```diff
@@ -101,19 +105,17 @@ const matchPath = (
   const allParamTypes = { ...coreParamTypes, ...paramTypes }
   let typeConstrainedRoute = route
 
-  // Map all params from the route to their type constraint regex to create a "type-constrained route" regexp
-  for (const [name, type] of routeParams) {
+  // Map all params from the route to their type constraint regex to create a
+  // "type-constrained route" regexp
```

Comment:
/regexp?/

<sub>(There's one "regex" and one "regexp" in the comment)</sub>

Reply:
I will randomly pick one!